src/index.html

```diff
@@ -146,9 +146,9 @@ <h2>Keep Your Flock Happy and Healthy</h2>
 <!-- GALLERY -->
 <section id="image-gallery" class="image-gallery dark-background full-width">
   <div class="gallery-content">
-    <img id="arrow-left" src="/imgs/prev-purple.svg" class="clickable" alt="Left scroll arrow" />
+    <img id="arrow-left" src="/imgs/prev-purple.svg" class="clickable" alt="Left scroll arrow" tabindex="0" />
```

Comment:
Not sure these need the `tabindex`? Images by themselves shouldn't be accessible via tabbing.

Reply:
@philiprenich it doesn't, I was testing the gallery for accessibility and there is no way to navigate through the images with the keyboard but it looks like it needs to be re-designed to be more accessible